app/models.py

```diff
@@ -98,8 +98,8 @@ def update_from_original(self, original):
 
 class ServiceUser(db.Model):
     __tablename__ = 'user_to_service'
-    user_id = db.Column(UUID(as_uuid=True), db.ForeignKey('users.id'), primary_key=True)
-    service_id = db.Column(UUID(as_uuid=True), db.ForeignKey('services.id'), primary_key=True)
+    user_id = db.Column(UUID(as_uuid=True), db.ForeignKey('users.id'), primary_key=True, nullable=True)
+    service_id = db.Column(UUID(as_uuid=True), db.ForeignKey('services.id'), primary_key=True, nullable=True)
```

Comment:
I think having a nullable primary key will cause integrity errors.

Reply:
Nullable was changed to "True" in the model to match the `user_id `and `service_id `column type we have in the database